README.md

```diff
@@ -81,6 +81,46 @@ end
 `text` fields will be full-text searchable. Other fields (e.g.,
 `integer` and `string`) can be used to scope queries.
 
+### Nested Documents support
+
+You can use the `child_documents` feature to add nested documents to other models,
+as such:
+
+```ruby
+class Comment < ActiveRecord::Base
+  searchable do
+    # NOTE: this is not necessary, Solr uses '_root_' field to refer to the parent document
+    integer :post_id
+    # -------------------------------------------------------------------------------------
+    integer :author_id
+    
+    text :body
+    time :published_at
+  end
+end
+
+class Post < ActiveRecord::Base
+  searchable do
+    text :title, :body
+    child_documents :comments # Must be of type Comment
+    
+    boolean :featured
+    integer :blog_id
+    integer :author_id
+    integer :category_ids, :multiple => true
+    double  :average_rating
+    time    :published_at
+    time    :expired_at
+    
+    string  :sort_title do
+      title.downcase.gsub(/^(an?|the)/, '')
+    end
+  end
+end
+```
+
+Please note, you should always use an `Array` of `searchable` documents in the `child_documents` field.
```

Comment:
is there a safe-check for the `searchable` usage in the DSL itself?
I would suggest moving this line to a line comment above